packages/zoe/test/unitTests/contracts/test-sellTickets.js

```diff
@@ -243,6 +243,13 @@ test(`mint and sell opera tickets`, async t => {
       aliceProposal,
       alicePaymentKeywordRecord,
     );
+
+    const offerResult = await E(seat).getOfferResult();
+    t.is(
+      offerResult,
+      'The offer has been accepted. Once the contract has been completed, please check your payout',
+    );
```

Comment:
This would have been a valid addition before this PR, yes?

Reply:
Yes! I needed this for debugging and thought it would be useful in the future so I left it in